### DISCUSSION TITLE
Oculars plugin: Eyepieces and FOV

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2074] [Q&A]
Stellarium asks Field Stop number to be filled in the New eyepiece form. But usually only focal lenght and apparent FOV are provided by manufacturers. As a matter of fact, FOV is usually calculated using the simplified expression Eyepiece Field of View  / (Telescope Focal Length / Eyepiece Focal Length) that doesn't use Field Stop number at all. But still, Stellarium asks for it. 

One way to supply it is just calculating the Field Stop _d_ out of the usual formulae with fT and fE focal lenghts of telescope and eyepiece respectively:

FOV  =2ATAN[d/(2fT)] so d=2fT TAN(FOV/2)
aFOV=2ATAN[d/(2fE)] so d=2fE TAN(aFOV/2)

Would it be nice if the plugin oculars could help calculate the missing Field Stop?

**Comment by alex-w** [comment]
Field stop number is optional. Please read the Stellarium User Guide.
